var/spack/repos/builtin/packages/r-globaloptions/package.py

```diff
@@ -10,12 +10,15 @@ class RGlobaloptions(RPackage):
     """It provides more controls on the option values such as validation and
        filtering on the values, making options invisible or private."""
 
-    homepage = "https://cran.r-project.org/package=GlobalOptions"
-    url      = "https://cran.rstudio.com/src/contrib/GlobalOptions_0.0.12.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/GlobalOptions"
+    homepage = "https://cloud.r-project.org/package=GlobalOptions"
+    url      = "https://cloud.r-project.org/src/contrib/GlobalOptions_0.0.12.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/GlobalOptions"
 
+    version('0.1.0', sha256='567a0a51f6b7b14127302f00e6e4944befd4964c069f96a9e61256e8c3c79ef2')
     version('0.0.12', '6c268b3b27874918ba62eb0f6aa0a3e5')
 
+    depends_on('r@2.10:', when='@:0.0.12', type=('build', 'run'))
+    depends_on('r@3.3.0:', when='@0.0.13:', type=('build', 'run'))
     depends_on('r-testthat', type=('build', 'run'))
     depends_on('r-knitr', type=('build', 'run'))
     depends_on('r-markdown', type=('build', 'run'))
```

Comment:
These 3 are suggested, not required. Removed

Reply:
Done.